src/status_im/contexts/chat/messenger/composer/events.cljs

```diff
@@ -140,15 +141,38 @@
           preferred-name (get-in db [:profile/profile :preferred-name])
           emoji? (emoji-only-content? {:text        input-text
                                        :response-to message-id})]
-      {:chat-id       current-chat-id
-       :content-type  (if emoji?
-                        constants/content-type-emoji
-                        constants/content-type-text)
-       :text          input-text
-       :response-to   message-id
-       :ens-name      preferred-name
-       :link-previews (map #(select-keys % [:url :title :description :thumbnail])
-                           (get-in db [:chat/link-previews :unfurled]))})))
+      {:chat-id              current-chat-id
+       :content-type         (if emoji?
+                               constants/content-type-emoji
+                               constants/content-type-text)
+       :text                 input-text
+       :response-to          message-id
+       :ens-name             preferred-name
+       :link-previews        (->> (get-in db [:chat/link-previews :unfurled])
+                                  (map #(select-keys %
+                                                     [:url :title :description :thumbnail
+                                                      :status-link-preview?]))
+                                  (filter (fn [preview]
+                                            (not (:status-link-preview? preview)))))
+       :status-link-previews (map (fn [{{:keys [community-id color description display-name
```

Comment:
why do we build status link here ?

Reply:
Because status-go expects the body of the message to send to have status-link-previews attached to it, Just like we do for normal link previews.

Reply:
but why, what's the point? for the normal link, we do it because of privacy, but why we want this for status link?

Reply:
This is done :
1. For sake of consistency
2. Enable to immediately show the contact/community/channel preview to the recipient. Not to wait while it's being fetched from waku.